### PR TITLE
Add in Nunavut scraper

### PR DESCRIPTION
**Pull request by seamuslee001**
This is dependent on https://github.com/opencivicdata/ocd-division-ids/pull/374

### CODE REVIEW

File: ca_nu/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from utils import CanadianJurisdiction
+
+
+class Nunavut(CanadianJurisdiction):
+    classification = "legislature"
+    division_id = "ocd-division/country:ca/territory:nu"
+    division_name = "Nunavut"
+    name = "Nunavut Legislative Assembly"
+    url = "https://www.assembly.nu.ca"
+    parties = []
```
Comment (jpmckinney) [inline ca_nu/__init__.py:10]:
```suggestion
```

File: ca_nu/people.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+class NunavutPersonScraper(CanadianScraper):
+    def scrape(self):
+        page = self.lxmlize(COUNCIL_PAGE)
+
+        members = page.xpath('//*[@id="content"]/section/div/div[3]/div/div[2]/div')
+        assert len(members), "No members found"
+        for member in members:
+            if "Vacant" not in member.xpath("./span[2]")[0].text_content():
```
Comment (jpmckinney) [inline ca_nu/people.py:14]: To reduce indentation prefer this style:

```python
            if "Vacant" in member.xpath("./span[2]")[0].text_content():
                continue

            ...
```

**Comment by seamuslee001** [comment]
@jpmckinney thanks James, I have updated this now

**Comment by jpmckinney** [comment]
FYI, this can't be imported into Represent until you find a shapefile (or other GIS file) for their constituencies.

**Pull request merged**